tardis/montecarlo/montecarlo_numba/formal_integral.py

```diff
@@ -416,8 +419,9 @@ def make_source_function(self):
         Numpy array containing ( 1 - exp(-tau_ul) ) S_ul ordered by wavelength of the transition u -> l
         """
 
-        simulation_state = self.model
+        simulation_state = self.simulation_state
         transport = self.transport
+        mct_state = transport.transport_state
```

Comment:
I assume ``mct_state`` is short for ``montecarlo_state``? I would prefer an explicit variable name to avoid confusion and to be coherent with the other changes, but since this is somewhat self-contained it might be ok.

Reply:
Agreed regarding a stronger variable name

Reply:
`montecarlo_transport_state` ?

Reply:
done!